src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/KubernetesPipelineTest.java

```diff
@@ -468,4 +468,22 @@ public void runInPodWithRetention() throws Exception {
         r.assertBuildStatusSuccess(r.waitForCompletion(b));
         assertTrue(deletePods(cloud.connect(), getLabels(this, name), true));
     }
+
+    @Test
+    public void runInPodValidateOverrideSlaveConnectTimeout() throws Exception {
+        System.setProperty(org.csanchez.jenkins.plugins.kubernetes.PodTemplate.class.getName()+".connectionTimeout", "10");
```

Comment:
you need to clear the property after the test or it will affect other tests

you could just do this test as part of another test to avoid increasing the test time during build

Reply:
Cool. I'm iterating a few times as I was not aware of the ITs setup :(